controllers/front/GuestTrackingController.php

```diff
@@ -91,10 +92,16 @@ public function postProcess()
                     [],
                     'Shop.Forms.Help'
                 );
-            } elseif (strlen($password) < Validate::PASSWORD_LENGTH) {
+            } elseif (Validate::isAcceptablePasswordLength($password)) {
```

Comment:
Am I misunderstanding something here? 🤔
```suggestion
            } elseif (!Validate::isAcceptablePasswordLength($password)) {
```